get-started/versioning-availability.md

```diff
@@ -4,168 +4,209 @@ mapped_pages:
   - https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/introducing-elastic-documentation.html
 products:
   - id: elastic-stack
+description: Learn how Elastic handles versioning and feature availability in the docs. Find the product versions that are supported, how to read availability badges, and...
 ---
 
-# Understanding versioning and availability
+# Versioning and availability in Elastic Docs
 
-On this page, you'll learn about our versioning systems, how to understand how we talk about versions and feature availability in our documentation, and where to find docs for your product version.
+Learn how Elastic Docs handles versioning, feature availability, and how to find the right documentation for your deployment type and product version. Find answers to common questions about the Elastic Stack versioning and to help you confidently navigate our continuously updated documentation.
 
-## Elastic Stack versioning
+## Frequently asked questions
 
-{{es}} and the core components of the Elastic Stack use a semantic versioning scheme. This scheme consists of three numbers separated by periods in the form `X.Y.Z`, for example: `9.0.0`.
+### Where can I find documentation for the latest version of the {{stack}}?
 
-Each number represents a specific level of change:
+You’re in the right place! All documentation for Elastic Stack 9.0.0 and later is available at [elastic.co/docs](https://www.elastic.co/docs), including the latest 9.1.0 version and any future versions in the 9.x series.
```

Comment:
We should use something like `{{version.stack| M.M}}` here so we don't have to update every release.

```suggestion
You’re in the right place! All documentation for Elastic Stack 9.0.0 and later is available at [elastic.co/docs](https://www.elastic.co/docs), including the latest {{version.stack| M.M}} version and any future versions in the 9.x series.
```